src/public/orderNotifications.js

```diff
@@ -0,0 +1,15 @@
+const io = require('socket.io-client');
```

Comment:
`require` is only available within node.js files themselves, this Javascript file is not running on our node server but rather in the client/browser. 

The socket.io server-side npm package, automatically creates a public/static path for the socket.io-client: [https://socket.io/docs/v4/client-installation/#:~:text=io%20will%20be%20registered%20as%20a%20global%20variable%3A](https://socket.io/docs/v4/client-installation/#:~:text=io%20will%20be%20registered%20as%20a%20global%20variable%3A)

That is how we were able to import it in the HTML: `<script src="/socket.io/socket.io.js"></script>`

Now that `io` variable is available globally in any script loaded by this HTML. So we don't need to try to require or import anything, we can go ahead and use it right away.

If you get a lint error because ESLint doesn't realize that `io` will be available globally (or any other third-party scripts you add to the HTML document), then you can add a comment at the top like this: `/* global io */`